packages/slice-machine/tests/__mocks__/fetchMock.ts

```diff
@@ -0,0 +1,17 @@
+export function FetchMock(
+  expectedUrl: string,
+  status: number = 200,
+  defaultValue: Record<string, unknown> = {}
+) {
+  global.fetch = jest.fn((url: RequestInfo | URL) => {
```

Comment:
interesting, usually I would use `nock` or `msw` to intercept the network responses.

Reply:
Actually it's because I got an error saying "fetch is not defined".
I used fetch in the code because I find it simple, non-library-dependent.

It's just when the test ran, it would just break 😅, so I made a helper to mock jest.